src/QuokkaSimulation.hpp

```diff
@@ -1675,9 +1677,12 @@ void QuokkaSimulation<problem_t>::subcycleRadiationAtLevel(int lev, amrex::Real
 					    static_cast<double>(global_iteration_sum) / static_cast<double>(global_solver_count);
 					const double global_solving_mean =
 					    static_cast<double>(global_solver_count) / static_cast<double>(n_cells) / 2.0; // 2 stages
-					amrex::Print() << "average number of Newton-Raphson solvings per IMEX stage is " << global_solving_mean
-						       << ", (mean, max) number of Newton-Raphson iterations are " << global_iteration_mean << ", "
-						       << global_iteration_max << "\n";
+					const double global_decoupled_iteration_mean =
+					    static_cast<double>(global_decoupled_iteration_sum) / static_cast<double>(global_solver_count);
+					amrex::Print()
+					    << "average number of Newton-Raphson solvings per IMEX stage is " << global_solving_mean
+					    << ", (mean, max) number of Newton-Raphson iterations are " << global_iteration_mean << ", " << global_iteration_max
```

Comment:
Printing this information out in a system that is not using decoupled dust will be confusing. Can we wrap this in an appropriate constexpr'ed if?

Reply:
Done.